pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -123,18 +130,63 @@ func verifyDevicePath(diskUUID string) string {
 }
 
 func probeAttachedVolume() error {
-	executor := exec.New()
-	args := []string{"trigger"}
-	cmd := executor.Command("/usr/bin/udevadm", args...)
-	_, err := cmd.CombinedOutput()
+	var scsiPath = "/sys/class/scsi_host/"
+	var b bytes.Buffer
+	var c bytes.Buffer
+	var driverType = "mpt"
+	var scanVal = []byte("- - -")
+
+	files, err := ioutil.ReadDir(scsiPath)
 	if err != nil {
-		glog.Errorf("error running udevadm trigger %v\n", err)
-		return err
+		glog.V(2).Infof("Error scanning attached vSphere volumes: %v", err)
+		return ErrProbeVolume
+	}
+	for _, file := range files {
+		b.WriteString(scsiPath)
+		b.WriteString(file.Name())
+		b.WriteString("/proc_name")
+		driverFile := b.String()
+
+		// grep for 'mpt' in 'proc_name'
```

Comment:
- Why udev trigger stop working?
- Is there any vSphere doc pointing to why only `mpt` is in `proc_name`?